### PR TITLE
feat(config): default `dockerImagePrefix` to `containerbase`

### PR DESCRIPTION
**Pull request by viceice**
## Changes
Change `dockerImagePrefix` from `renovate` to `containerbase`.
Probably breaking so some users.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [x] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/self-hosted-configuration.md [hunk 1]
```diff
@@ -313,19 +313,19 @@ For example, if you set `dockerChildPrefix=myprefix_` then the final container c
 
 ## dockerImagePrefix
 
-By default Renovate pulls the sidecar Docker containers from `docker.io/renovate`.
+By default Renovate pulls the sidecar Docker containers from `docker.io/containerbase`.
```
Comment (rarkins) [inline docs/usage/self-hosted-configuration.md:316]: Is shifting to ghcr in future also a possibility?

Comment (viceice) [reply docs/usage/self-hosted-configuration.md:316]: yes, the containerbase images are also published to ghcr.io

Comment (rarkins) [reply docs/usage/self-hosted-configuration.md:316]: Probably best to keep it with Docker Hub for now though as we know they work perfectly well until now (rate limits aside)

Comment (viceice) [reply docs/usage/self-hosted-configuration.md:316]: i can add a note that you can also use the ghcr for official sidecar image

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

**Review by HonkingGoose — APPROVED** [review]
I'm happy with the docs.

**Review by rarkins — APPROVED** [review]

**Pull request merged**